decidim-comments/spec/cells/decidim/comments/comment_thread_cell_spec.rb

```diff
@@ -19,31 +19,6 @@ module Decidim::Comments
       it "renders the thread" do
         expect(subject).to have_css(".comment-thread")
         expect(subject).to have_content(comment.body.values.first)
-
-        expect(subject).not_to have_css(".comment-thread__title")
-      end
-
-      context "with replies" do
```

Comment:
Are we sure we want to remove those specs ?

Reply:
I removed them because the title is not present in the redesign, but you are right I've updated the selectors to test this behaviour in a different way.